obpa/readme.md

```diff
@@ -0,0 +1,22 @@
+---
+noteId: "609285208aa711ed902cd7b714a6c555"
+tags: []
+
+---
+
+# The ontology for building permit authorities provides a vocabulary to describe the organization of building authorities.
```

Comment:
```suggestion
# OBPA — The Ontology for Building Permit Authorities
```

Reply:
@dgarijo @TallTed thanks for your support. The suggestion is fine!